READMEs/README_zh-CN.md

```diff
@@ -0,0 +1,77 @@
+# ReZygisk
+
+[Bahasa Indonesia](/READMEs/README_id-ID.md)|[Tiếng Việt](/READMEs/README_vi-VN.md)|[Português Brasileiro](/READMEs/README_pt-BR.md)|[French](/READMEs/README_fr-FR.md)|[日本語](/READMEs/README_ja-JP.md)|[中文简体](/READMEs/README_zh-CN.md)
```

Comment:
You shouldn't put the entire list in the translation READMEs. See other translations.